src/styles/card.scss

```diff
@@ -0,0 +1,128 @@
+.products {
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+
+  gap: 30px 48px;
+  font-family: Roboto, sans-serif;
```

Comment:
Font-family should be defined in the root or body selector for the whole project, not within individual components, to maintain consistency across the website.